astropy/io/votable/exceptions.py

```diff
@@ -31,25 +31,36 @@
 {exceptions}
 """
 
-
 # STDLIB
 import io
 import re
 
 from textwrap import dedent
 from warnings import warn
 
+from astropy import config as _config
 from astropy.utils.exceptions import AstropyWarning
 
-
 __all__ = [
-    'warn_or_raise', 'vo_raise', 'vo_reraise', 'vo_warn',
+    'Conf', 'conf', 'warn_or_raise', 'vo_raise', 'vo_reraise', 'vo_warn',
```

Comment:
Is the capital C `Conf` needed here?  I didn't notice it being referenced, but I'm probably missing some of the black magic needed for setup.

Reply:
I just copy-pasted that part from the `__init__.py` file. If you feel strongly about it, I can remove it. I think all this affects is the `from astropy.io.votable.exceptions import *` command and also when Sphinx scrapes the files for API doc.

Reply:
Ah, actually, I do need to change the doc a little to make it render.

Reply:
I pushed the doc update. Given that the doc doesn't use `automodapi` for `exceptions`, this addition only affects star-imports. Are you okay with leaving this in or you want me to take it back out? Please advise, @tomdonaldson . Thanks!

Reply:
I'm fine leaving it in since I'm not sure enough that it would never be useful.  It's certainly not harmful in any important way.